server/repository/src/db_diesel/sync_file_reference_row.rs

```diff
@@ -31,6 +54,26 @@ pub struct SyncFileReferenceRow {
     pub record_id: String,
     pub file_name: String,
     pub mime_type: Option<String>,
+    #[serde(skip_serializing)]
```

Comment:
All these 
```
    #[serde(skip_serializing)]
    #[serde(default)]
```
Records are local only state, so not included in the sync record.
Feels a bit weird that that logic is here in the repository layer but using serde does make this easy...